### PR TITLE
fix: Add missing content type to controllers (no-changelog)

### PR DESCRIPTION
**Pull request by krynble**
The lack of a proper content-type was causing Firefox to believe the request was XML and therefore print out an unnecessary log

Untitled (1)

### CODE REVIEW

File: packages/cli/src/credentials/credentials.controller.ee.ts [hunk 1]
```diff
@@ -139,7 +144,7 @@ EECredentialsController.put('/:credentialId/share', async (req: CredentialReques
 	const { shareWithIds } = req.body;
 
 	if (!Array.isArray(shareWithIds) || !shareWithIds.every((userId) => typeof userId === 'string')) {
-		return res.status(400).send('Bad Request');
+		return res.status(400).send({ message: 'Bad request' });
```
Comment (netroy) [inline packages/cli/src/credentials/credentials.controller.ee.ts:147]: you can change this to use `.json` instead, as `.send` expects a string. then you don't need to add `Content-Type` explicitly.

```suggestion
		return res.status(400).json({ message: 'Bad request' });
```

**Comment by krynble** [comment]
Thanks @netroy I've realized that simply switching to use `ResponseHelper.send` is a better way of solving the problem. It was missing in just a couple places.

**Review by netroy — APPROVED** [review]

File: packages/cli/src/credentials/credentials.controller.ee.ts [hunk 2]
```diff
@@ -134,50 +134,56 @@ EECredentialsController.post(
...
+	'/:credentialId/share',
+	ResponseHelper.send(async (req: CredentialRequest.Share, res) => {
+		const { credentialId } = req.params;
+		const { shareWithIds } = req.body;
+
+		if (
+			!Array.isArray(shareWithIds) ||
+			!shareWithIds.every((userId) => typeof userId === 'string')
+		) {
+			return res.status(400).send({ message: 'Bad request' });
```
Comment (netroy) [inline packages/cli/src/credentials/credentials.controller.ee.ts:147]: This still needs to be
```suggestion
			return res.status(400).json({ message: 'Bad request' });
```

**Review by netroy — CHANGES_REQUESTED** [review]

File: packages/cli/src/credentials/credentials.controller.ee.ts (hunk 2 above)
Comment (netroy) [reply packages/cli/src/credentials/credentials.controller.ee.ts:147]: nevermind. [looks like like `.send` now internally calls `.json` for objects.](https://github.com/expressjs/express/blob/master/lib/response.js#L162)

File: packages/cli/src/credentials/credentials.controller.ee.ts [hunk 3]
```diff
@@ -134,50 +134,56 @@ EECredentialsController.post(
  * Grant or remove users' access to a credential.
  */
 
-EECredentialsController.put('/:credentialId/share', async (req: CredentialRequest.Share, res) => {
-	const { credentialId } = req.params;
-	const { shareWithIds } = req.body;
+EECredentialsController.put(
+	'/:credentialId/share',
+	ResponseHelper.send(async (req: CredentialRequest.Share, res) => {
```
Comment (netroy) [inline packages/cli/src/credentials/credentials.controller.ee.ts:139]: looks like we can't mix `ResponseHelper.send` and `res.send` as they each try to set their own set of headers, which fails.

Comment (krynble) [reply packages/cli/src/credentials/credentials.controller.ee.ts]: Yup, I messed up, pushing new changes now

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.203.0`

**Pull request merged**